### ISSUE TITLE
Text Classification / Text Categorization.

### ISSUE THREAD

**Issue by Sana-72** [AldoStalin/SonaITHackathon#27]
WORD SPEAKS - Text Classification / Text Categorization

* Project Name:  Text Classification / Text Categorization.

* Short Project Description: In this project,we propose a text categorization approach and this paper illustrates the text
                                            classification process using machine learning algorithms.

* Team Name: WORD SPEAKS.

* Team Members:  Santhini B  @Sana72 , Srilekha P @Srilekha-10 , Ravi Shankar S @Ravishankar6

* Demo Link:  https://youtu.be/t0pcXsaTmmU

* Repository Link: https://github.com/Sana-72/Mini-project-2-Text-Categorization

* Labels: Supervised learning,Machine learning.

* Pitch
        *   Task of assigning predefined categories to documents.Process of assigning categories to the text according to the 
            content. Our project is in supervised learning.In supervised learning we have labeled data.
        *  Classifier will take this text (Dataset) as an input and will analyse its content and automatically assigned relevant tags.This 
            type of categorizing text can be done in two ways.Manually,automatic classification.For that there are some popular 
            machine learning algorithms are these for creating text classification model-  SVM, DL.
        *  Data Preparation:
                        1. First we will split the data intro training and testing.
                        2. Then we will tokenize the text by using label encoder.
                        3. Then we will convert the labels to one-hot representation .
        *  Train : Now we are building the model by running epochs on train sample.
        *  Evaluate : Here our accuracy of the model is 95%.
        *  Hyperparameter :  It controls the overall behaviour of the model. It ultimate goal is to find optional combination it 
                                          minimizes the predefined loss function to give better results.
